modules/oxide/src/less/theme/components/dialog/dialog.less

```
@@ -363,6 +363,12 @@
     max-width: 1200px;
   }
 
+  .tox-dialog--width-lg--inline {
```

Small nitpick. The existing dialog inline class is  `.tox-dialog-inline {`
For consistency, I wonder whether it should be
```suggestion
  .tox-dialog-inline--width-lg {
```